README.md

```
@@ -1 +1,26 @@
-# reto-winter-2025-google
\ No newline at end of file
+# reto-winter-2025-google
+
+### Docs for devs
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)